src/models/parallelCoordinates.js

```diff
@@ -100,7 +100,7 @@ nv.models.parallelCoordinates = function() {
                         max = y[d].domain()[1];
                     }
                         //If the brush extent is > max (< min), keep the extent value.
-                    else if (!f[0].hasNaN && displayBrush) {
+                    else if (!f[0].hasOnlyNaN && displayBrush) {
```

Comment:
HasOnlyNan is already existing, in fact when I commit this code in a first place I made a typo on setting hasNaN, here we need to check hasOnlyNaN. This check if there is only nan values on the brush (so the brush is below the dimension axis, and on the undefined values line).